imgee/storage.py

```diff
@@ -29,17 +29,23 @@ def get_resized_image(img, size, is_thumbnail=False):
     """
     registry = imgee.registry
     img_name = img.name
+
+    if img.mimetype == 'image/gif':
+        # if the gif file is animated, no need to resize
+        src_path = download_from_s3(img.name + img.extn)
```

Comment:
It does include the period. and the period is stored in db too. and `.extn` is a function property https://github.com/hasgeek/imgee/blob/master/imgee/models/stored_file.py#L58